project 3/CODE/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+    background: linear-gradient(to bottom, hsla(281, 100%, 25%, 1) 0%, hsla(22, 100%, 63%, 1) 51%, hsla(21, 97%, 63%, 1) 52%, hsla(281, 100%, 25%, 0.97) 100%);
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#570080', endColorstr='#570080', GradientType=0);
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin: 0;
+    padding: 10px;
+}
+
+h2 {
+    font-family: 'Ubuntu', sans-serif;
+}
+
+p {
+    font-family: 'Quicksand', sans-serif;
+    font-size: 20px;
+}
+
+.button-close,
+div h2,
+div p {
+    display: none;
+}
+
+#hour6am:target h2,
```

Comment:
See previous note in the HTML about simplifying this code and keeping it from being so repetitive.